dump/forum.sql

```diff
@@ -303,6 +303,31 @@ INSERT INTO `qa_eventlog` VALUES ('2016-05-16 19:53:27','192.168.112.1',1,'test'
 /*!40000 ALTER TABLE `qa_eventlog` ENABLE KEYS */;
 UNLOCK TABLES;
 
+--
+-- Table structure for table `qa_flagreasons`
+--
+
+DROP TABLE IF EXISTS `qa_flagreasons`;
```

Comment:
Nie umiem w bazy danych, więc zapytam - co ma na celu usuwanie tabeli dla powodów zgłoszeń, by poniżej ją dodać z powrotem?

Reply:
To wygenerowany sql, nie dodawałem tego sam. A na celu ma zapewne to, aby najpierw wyrzucić tabelę gdyby już była i zrobić ją wg podanej tu instrukcji.